packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -314,6 +313,81 @@ contract OptimismPortal_Test is Portal_Initializer {
         vm.expectRevert("L2OutputOracle: No output found for that block number.");
         assertEq(op.isBlockFinalized(checkpoint + 1), false);
     }
+
+    function test_finalizeWithdrawalTransaction_differential(
+        address _sender,
+        address _target,
+        uint64 _value,
+        uint8 _gasLimit,
+        bytes memory _data
+    ) external {
+        uint256 _nonce = messagePasser.nonce();
+
+        (
+            bytes32 stateRoot,
+            bytes32 storageRoot,
+            bytes32 outputRoot,
+            bytes32 withdrawalHash,
+            bytes memory withdrawalProof
+        ) = ffi.finalizeWithdrawalTransaction(
```

Comment:
IMO this name should be changed to something like `ffi.getFinalizeWithdrawalTransactionInputs` (or less cumbersome-ly `ffi.getFinalizationInputs`). Most of the other `case`s in `differential-testing.ts` are a direct reimplementation of the solidity functions they are named after.